src/test/java/uk/gov/hmcts/reform/sendletter/services/PendingLettersServiceTest.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.sendletter.services;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import uk.gov.hmcts.reform.sendletter.entity.Letter;
+import uk.gov.hmcts.reform.sendletter.entity.LetterRepository;
+import uk.gov.hmcts.reform.sendletter.entity.LetterStatus;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+class PendingLettersServiceTest {
+
+    @Test
+    void should_read_letters_in_proper_status_from_repo() {
+        // given
+        LetterRepository repo = mock(LetterRepository.class);
+        PendingLettersService service = new PendingLettersService(repo);
+        List<Letter> noLetters = Collections.emptyList();
```

Comment:
Empty list is a good case to test 👍. It would also be useful to have a test for a non-empty list. Alternatively, if we just care if the service has returned what it got the repository, we can create an assertion for exactly the same instance.

Reply:
did not want to tamper too much with existing pr but can include extra test just for the sake of validity